test_tf2/test/test_message_filter.cpp

```diff
@@ -268,7 +268,7 @@ TEST(MessageFilter, outTheBackFailure)
 
   builtin_interfaces::msg::Time stamp(1);
   bc.setTransform(createTransform(Quaternion(0,0,0,1), Vector3(1,2,3), stamp, "frame1", "frame2"), "me");
-  bc.setTransform(createTransform(Quaternion(0,0,0,1), Vector3(1,2,3), stamp + tf2::Duration(10000), "frame1", "frame2"), "me");
+  bc.setTransform(createTransform(Quaternion(0,0,0,1), Vector3(1,2,3), stamp + tf2::durationFromSec(10000), "frame1", "frame2"), "me");
```

Comment:
if you're wondering if this was already ns, upstream it is also 10,000(s): https://github.com/ros/geometry2/blob/439e2350ca9b2f0581a3631f0929acc2021db73e/test_tf2/test/test_message_filter.cpp#L272